tests/system/libraries/NvdaLib.py

```diff
@@ -51,7 +51,7 @@ def __init__(self):
 		self.whichNVDA = builtIn.get_variable_value("${whichNVDA}", "source")
 		if self.whichNVDA == "source":
 			self._runNVDAFilePath = _pJoin(self.repoRoot, "source/nvda.pyw")
-			self.baseNVDACommandline = f"pyw -3.7-32 {self._runNVDAFilePath}"
+			self.baseNVDACommandline = f"pyw {self._runNVDAFilePath}"

```

Comment:
Can't you just use runnvda.bat here?

Reply:
Yes, running the bat file might be better. It would be good to centralize the method used to invoke python. I've run into nasty issues before where a different version of python was used to execute the system tests. Hence the `-3.7-32` argument.

Reply:
I don't think you've addressed this one.